x-pack/plugins/security/server/routes/api_keys/privileges.test.ts

```diff
@@ -70,16 +109,21 @@ describe('Check API keys privileges', () => {
 
     const error = Boom.notAcceptable('test not acceptable message');
     getPrivilegesTest('returns error from cluster client', {
-      apiResponses: [
+      callAsCurrrentUserResponses: [
         async () => {
           throw error;
         },
-        async () => {},
       ],
+      callAsInternalUserResponses: [async () => {}],
       asserts: {
-        apiArguments: [
-          ['shield.hasPrivileges', { body: { cluster: ['manage_security', 'manage_api_key'] } }],
-          ['shield.getAPIKeys', { owner: true }],
+        callAsCurrentUserAPIArguments: [
+          [
+            'shield.hasPrivileges',
+            { body: { cluster: ['manage_security', 'manage_api_key', 'manage_own_api_key'] } },
+          ],
+        ],
+        callAsInternalUserAPIArguments: [
```

Comment:
My only motivation here was to follow the pattern that already existed in this test suite. The changes started out small, but it might have been easier to rewrite in hindsight

Reply:
Got it, makes sense to me.